frontend/src/metabase/common/components/AggregationPicker/AggregationPicker.tsx

```diff
@@ -329,9 +328,11 @@ export function AggregationPicker({
     const columns = Lib.aggregationOperatorColumns(operator);
     const columnGroups = Lib.groupColumns(columns);
     return (
-      <ColumnPickerContainer
+      <Box
         className={className}
+        mih="18.75rem"
         data-testid="aggregation-column-picker"
+        c="summarize"
```

Comment:
adding this one so we can remove the `AggregationPicker` styled component [here](https://github.com/metabase/metabase/pull/49836/files#diff-4c9ff34ad6986aa3a77066da25a82f3deff46b07c873cacea39dc89173f97c4eL30)